src/html/create_songs_html.py

```diff
@@ -0,0 +1,192 @@
+        _add_creator(song.original_title, "Tytuł oryginalny: ", body_song)
+    if song.alias:
+        _add_creator(song.alias, "Tytuł alternatywny: ", body_song)
+    if song.text_author:
+        _add_creator(song.text_author, "Słowa: ", body_song)
+    if song.translator:
+        _add_creator(song.translator, "Tłumaczenie: ", body_song)
+    if song.composer:
+        _add_creator(song.composer, "Muzyka: ", body_song)
+    if song.music_source:
+        _add_creator(song.music_source, "Melodia oparta na: ", body_song)
+    if song.artist:
+        _add_creator(song.artist, "Wykonawca: ", body_song)
+    if song.album:
+        _add_creator(song.album, "Album: ", body_song)
+    if song.metre:
+        _add_creator(song.metre, "Metrum: ", body_song)
+    if song.barre and int(song.barre) > 0:
+        _add_creator(song.barre, "Kapodaster: ", body_song)
+    for block in song.blocks:
+        if block.block_type.value == 'V':
+            b_type = "verse"
+        elif block.block_type.value == 'C':
+            b_type = "chorus"
+        else:
+            b_type = "other"
+        _add_verse(block, body_song, b_type)
+    if song.comment:
+        div = etree.SubElement(body_song, "div", attrib={"class": "comment"})
+        span_content = etree.SubElement(div, "span", attrib={"class": "comment"})
+        span_content.text = song.comment
+
+
+def xml2html(src_xml_path, path_out):  # tworzy piosenkę w wersji html
+
+    xhtml_namespace = "http://www.w3.org/1999/xhtml"
+    xhtml = "{%s}" % xhtml_namespace
+    nsmap = {None: xhtml_namespace}
+
+    song = rsx.parse_song_xml(src_xml_path)
+    root_html = etree.Element(xhtml + "html", nsmap=nsmap)
+    head = etree.SubElement(root_html, "head")
+    etree.SubElement(head, "link",
+                     attrib={"rel": "stylesheet", "type": "text/css", "href": "CSS/song.css", "media": "screen"})
+    _add_blocks(song, root_html)
+    et = etree.ElementTree(root_html)
+    et.write(path_out, pretty_print=True, method='xml', encoding='utf-8', xml_declaration=True)
+
+
+def create_list_of_songs(song_set):
+    """ Dostaje jako argument listę piosenek lub ścieżkę do katalogu i zwraca listę piosenek """
+    if str(type(song_set)) == "<class 'list'>":
+        for i in range(len(song_set)):
+            if song_set[i][-4:] == '.xml':
+                song_set[i] = song_set[i][0:-4]
+        return song_set
+    else:
+        songs_list = os.listdir(song_set)
+        for i in range(len(songs_list)):
+            if songs_list[i][-4:] == '.xml':
+                songs_list[i] = songs_list[i][0:-4]
+        return songs_list
+
+
+def create_all_songs_html(list_of_songs, path_in, path_out):
+    """Tworzy wszystkie piosenki z listy w formacie html w katalogu path_out"""
+    list_of_songs = create_list_of_songs(list_of_songs)
+    for song in list_of_songs:
+        xml2html(os.path.join(path_in, song + '.xml'), os.path.join(path_out, song + '.html'))
+
+# to test
+# def main():
```

Comment:
Maybe uncomment, but call it "test()" instead of main() ?

Reply:
i delate all of comment